packages/cli/src/checkBeforeRunning.ts

```diff
@@ -0,0 +1,39 @@
+import {Hook} from '@oclif/config'
+import isBlitzRoot, {IsBlitzRootError} from './utils/is-blitz-root'
+import chalk from 'chalk'
+
+const whitelistGlobal = ['new']
+
+const hook: Hook<'init'> = async function (options) {
+  // Bug with oclif
```

Comment:
did you open an issue with their project? would be good to get this fixed at some point

Reply:
https://github.com/oclif/config/issues/108